elements/pfe-navigation/src/pfe-navigation-item.scss

```diff
@@ -0,0 +1,148 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: navigation-item;
+
+:host {
+  --pfe-navigation-item--Color: #{pfe-color(text--on-dark)};
+  --pfe-navigation-item--BackgroundColor: #{pfe-color(surface--darkest)};
+  --pfe-navigation-item--BorderTopColor: transparent;
+
+  --pfe-navigation-item__icon--Background: none;
+
+  height: 100%;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+}
+
+:host([pfe-icon]) {
+  display: flex;
+}
+
+@each $icon in (bento, user, globe, search, menu) {
+  :host([pfe-icon="#{$icon}"]) {
+    --pfe-navigation-item__icon--Background: #{map-get($icons, #{$icon}--on-dark)} no-repeat;
+  }
+  :host([pfe-icon="#{$icon}"].expanded) .pfe-navigation-item__trigger {
+    --pfe-navigation-item__icon--Background: #{map-get($icons, #{$icon})} no-repeat;
+  }
+}
+
+:host([hidden]),
+::slotted([slot="tray"][hidden]) {
+	display: none;
+}
+
+// Making the trigger styles an extend to optimize the css output below
+%trigger-styles {
+  font-family: #{pfe-var(font-family)};
+  text-align: center;
+  font-size: 14px;
+  font-weight: 100;
+  margin: 0;
+  white-space: nowrap;
+  color: #{pfe-local(Color)};
+  a {
+    cursor: pointer;
+    text-decoration: none;
+    color: inherit;
+  }
+  :host([pfe-icon]) & {
+    font-size: 12px;
+  }
+}
+
+#pfe-navigation-item--trigger {
+  @for $n from 1 through 6 {
+    h#{$n} {
+      @extend %trigger-styles;
+    }
+  }
+}
+
+::slotted([slot="tray"]) {
+  padding: 24px;
+  max-height: calc(100vh - var(--pfe-navigation--Height, 90px));
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
+
+.pfe-navigation-item {
+  &__icon {
+    display: flex;
+    background: var(--pfe-navigation-item__icon--Background);
+
+    :host(:not([pfe-icon])) [aria-expanded="false"] & {
+      visibility: hidden;
+    }
+
+    :host(:not([pfe-icon])) & {
+      @include pfe-arrow(down, gray, $width: 6px, $height: 6px);
+      position: absolute;
+      bottom: 13px;
+    }
+
+    :host([pfe-icon]) & {
+      margin-bottom: calc(#{pfe-var(container-spacer)} / 2);
+      width: 1.3em;
+      height: 1.3em;
+    }
+  }
+  &__trigger {
+    --pfe-theme--link--text-decoration: none;
+    --pfe-theme--link--text-decoration--hover: none;
+    --pfe-theme--link--text-decoration--focus: none;
+
+    --pfe-broadcasted--color--ui-link:          #{pfe-local(Color)};
+    --pfe-broadcasted--color--ui-link--hover:   #{pfe-local(Color)};
+    --pfe-broadcasted--color--ui-link--visited: #{pfe-local(Color)};
+    --pfe-broadcasted--color--ui-link--focus:   #{pfe-local(Color)};
+
+    transition: all #{pfe-var(animation-timing)};
+
+    position: relative;
+
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: center;
+    justify-content: center;
+    flex-grow: 1;
+
+    background-color: #{pfe-local(BackgroundColor)};
+    padding: 20px 16px !important;
```

Comment:
The mock-ups also show 20px of padding above and below the nav items so would need clarity on which is more important.

Reply:
Ah, the 20px spacer actually overlaps the icons if stretched across (will note this with Corey). Since the icon+links are the tallest items, we'll probably want to set padding based on those. 
![image](https://user-images.githubusercontent.com/456863/60460305-54a53100-9c11-11e9-9c6d-d34d6a71450c.png)

If I change the spacer to about 15px, and subtract the border-top of 4px, its about 10 or 11px.

Reply:
So update the nav-item padding top and bottom to....

Reply:
`padding: 10px 16px`

Reply:
What about `padding: 8px 16px`?  Based on the DSC chat, they are working towards all metrics being multiples of 8.

Reply:
So it would equate to: variable -> (container-padding / 2) and container-padding